ui/lib/replication/addon/templates/mode/secondaries/revoke.hbs

```diff
@@ -20,21 +20,20 @@
     The secondary id to revoke; given initially to generate a secondary token.
   </p>
 </div>
-<div class="field is-grouped box is-fullwidth is-bottomless">
-  <div class="control">
-    <ConfirmAction
-      @buttonText="Revoke"
-      @confirmTitle="Revoke token?"
-      @confirmMessage="This will revoke this secondary token."
-      @disabledMessage={{unless this.id "A secondary ID is required perform revocation."}}
-      @onConfirmAction={{action "onSubmit" "revoke-secondary" "primary" (hash id=this.id)}}
-    />
-  </div>
-  <div class="control">
-    {{#unless this.isRevoking}}
-      <LinkTo @route="mode.secondaries" @model={{this.replicationMode}} class="button">
-        Cancel
-      </LinkTo>
-    {{/unless}}
-  </div>
-</div>
\ No newline at end of file
+<hr class="has-background-gray-100" />
+<Hds::ButtonSet>
+  <ConfirmAction
+    @buttonText="Revoke"
+    @confirmTitle="Revoke token?"
+    @confirmMessage="This will revoke this secondary token."
+    @disabledMessage={{unless this.id "A secondary ID is required perform revocation."}}
+    @onConfirmAction={{action "onSubmit" "revoke-secondary" "primary" (hash id=this.id)}}
+  />
+  <Hds::Button
+    @text="Cancel"
+    @color="secondary"
+    @route="mode.secondaries"
+    @model={{this.replicationMode}}
+    @disabled={{not this.isRevoking}}
```

Comment:
I think this should be 
```suggestion
    @disabled={{this.isRevoking}}
```

Since before, `Cancel` rendered unless a revoke was in progress. So cancel is only allowed if a revoke is not running. So disabled should `true` if `this.isRevoking`